extensions/store/sql/identity-hub-store-sql/src/test/java/org/eclipse/edc/identityhub/store/sql/SqlIdentityHubStoreTest.java

```diff
@@ -0,0 +1,54 @@
+package org.eclipse.edc.identityhub.store.sql;
+
+import org.eclipse.edc.identityhub.store.spi.IdentityHubRecord;
+import org.eclipse.edc.identityhub.store.sql.schema.BaseSqlIdentityHubStatements;
+import org.eclipse.edc.spi.types.TypeManager;
+import org.eclipse.edc.transaction.datasource.spi.DefaultDataSourceRegistry;
+import org.eclipse.edc.transaction.spi.NoopTransactionContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.io.IOException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+public class SqlIdentityHubStoreTest {
+
+    @Mock
+    private ResultSet resultSet;
+
+    private SqlIdentityHubStore sqlIdentityHubStore;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        var statements = new BaseSqlIdentityHubStatements();
+        var typeManager = new TypeManager();
+        var dataSourceRegistry = new DefaultDataSourceRegistry();
+        var transactionContext = new NoopTransactionContext();
+
+        MockitoAnnotations.openMocks(this);
```

Comment:
not how we do mocks. please use the static `mock()` method. also, please use static imports.